birdhouse/pavics-compose.sh

```diff
@@ -150,6 +150,7 @@ do
     PROXY_SECURE_PORT=443 HOSTNAME=${PAVICS_FQDN} docker-compose ${COMPOSE_CONF_LIST} restart proxy
 
     # run postgres post-startup setup script
+    # Note: this must run before the post-docker-compose-up scripts since some may expect postgres databases to exist
```

Comment:
We still have an exception here humm.  Maybe if this scenario come back often, we can have `post-docker-compose-up-preparation` plugin entrypoint for this kind of scenario.